android_jni/avifandroidjni/build.gradle

```diff
@@ -0,0 +1,33 @@
+plugins {
```

Comment:
Might be worth looking at if this file is the source of the warning:

Deprecated Gradle features were used in this build, making it incompatible with Gradle 8.0.
Use '--warning-mode all' to show the individual deprecation warnings.
See https://docs.gradle.org/7.0.2/userguide/command_line_interface.html#sec:command_line_warnings

Reply:
Turns out this was just about having a jcenter() as a source for dependencies in settings.gradle (since that is being shut down).

I have removed it.